src/main/java/marquez/api/models/DatasetsResponse.java

```diff
@@ -14,14 +14,12 @@
 
 package marquez.api.models;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
 import java.util.List;
 import lombok.Data;
 import lombok.NonNull;
 
 @Data
 public final class DatasetsResponse {
   @NonNull
-  @JsonProperty("datasets")
   private final List<DatasetResponse> datasetResponses;
```

Comment:
Unfortunately, `@JsonProperty` is required here because we're telling jackson to override the name `datasetResponses` with `datasets` when serializing the class as Json. In order to remove `@JsonProperty`, let's rename the field as:

```
List<DatasetResponse> datasets;
```

Reply:
Fixed it.. Thanks for the help!!